docs/docs/upgrade/network-api-migration.md

```diff
@@ -0,0 +1,699 @@
+---
+title: Migrating to 1.5 Networking API
```

Comment:
it is -- my mistake. I decided it was more appropriate to an "Upgrading" section. I'll incorporate your comments and fix.